e2e/e2e_test.go

```diff
@@ -61,17 +61,19 @@ func TestE2EBasic(t *testing.T) {
 
 	runSyncStop(t, configPath, 20*time.Second)
 
-	resourcesPath := fmt.Sprintf("%s/%s", tempDir, resourcesDir)
-	files := testutils.CheckDir(t, true, resourcesPath)
-	require.Equal(t, 3, len(files))
```

Comment:
Could we check for the required number of files in each task's resource folder since this line has been removed?

Reply:
Good point! I added back the check for files in each of the new resource paths. Thanks!